src/main/java/uk/ac/ebi/ampt2d/accession/server/FileAccessioningController.java

```diff
@@ -27,20 +27,20 @@
 
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 import java.util.UUID;
 
 @RestController
 @RequestMapping(value = "/v1/accession")
-public class FileAccessioningServer {
+public class FileAccessioningController {
 
     @Autowired
     private UuidFileAccessioningService accessioningService;
 
     @RequestMapping(value = "/file", method = RequestMethod.POST, produces = "application/json",consumes = "application/json")
-    public FileAccessionResponse accessionFiles(@RequestBody List<UuidFile> files) {
+    public Set<UuidFile> accessionFiles(@RequestBody List<UuidFile> files) {
 
         Map<UuidFile, UUID> accessions = accessioningService.getAccessions(files);
```

Comment:
Given that the accession is an attribute of `UuidFile`, it there really a point on the service to return a map?

Reply:
If we return a map then it should be easy to generate a generic class to implement the different webservices to return a response.